Mac Big Sur 11.2 (intel) Error: You don't have permission to open the application "Neovide"

Hi, I've downloaded release '0.5.0'

and am seeing this error:
```
open Neovide.app
The application cannot be opened for an unexpected reason, error=Error Domain=NSOSStatusErrorDomain Code=-10826 "kLSNoLaunchPermissionErr: User doesn't have permission to launch the app (managed networks)" UserInfo={_LSFunction=_LSLaunchWithRunningboard, _LSLine=2508, NSUnderlyingError=0x7fa8bfd08cc0 {Error Domain=RBSRequestErrorDomain Code=5 "Launch failed." UserInfo={NSLocalizedFailureReason=Launch failed., NSUnderlyingError=0x7fa8bfd095a0 {Error Domain=NSPOSIXErrorDomain Code=111 "Unknown error: 111" UserInfo={NSLocalizedDescription=Launchd job spawn failed with error: 111}}}}}
```

<img width="372" alt="Screenshot 2021-02-08 at 10 21 42" src="https://user-images.githubusercontent.com/34974/107207372-23e5ca00-69f8-11eb-940a-5cdc2b0d1f46.png">

I get this error after @Benjamin-Davies's suggestion:

```
   Compiling skia-bindings v0.32.1
The following warnings were emitted during compilation:

warning: In file included from src/paragraph.cpp:10:
warning: /Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia/modules/skparagraph/include/ParagraphBuilder.h:19:44: warning: unused parameter 'style' [-Wunused-parameter]
warning:     ParagraphBuilder(const ParagraphStyle& style, sk_sp<FontCollection> fontCollection) { }
warning:                                            ^
warning: /Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia/modules/skparagraph/include/ParagraphBuilder.h:19:73: warning: unused parameter 'fontCollection' [-Wunused-parameter]
warning:     ParagraphBuilder(const ParagraphStyle& style, sk_sp<FontCollection> fontCollection) { }
warning:                                                                         ^
warning: In file included from src/paragraph.cpp:14:
warning: /Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia/modules/skparagraph/include/TypefaceFontProvider.h:47:42: warning: unused parameter 'index' [-Wunused-parameter]
warning:     SkFontStyleSet* onCreateStyleSet(int index) const override { return nullptr; }
warning:                                          ^
warning: /Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia/modules/skparagraph/include/TypefaceFontProvider.h:48:47: warning: unused parameter 'familyName' [-Wunused-parameter]
warning:     SkTypeface* onMatchFamilyStyle(const char familyName[],
warning:                                               ^
warning: /Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia/modules/skparagraph/include/TypefaceFontProvider.h:49:55: warning: unused parameter 'style' [-Wunused-parameter]
warning:                                    const SkFontStyle& style) const override {
warning:                                                       ^
warning: /Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia/modules/skparagraph/include/TypefaceFontProvider.h:52:56: warning: unused parameter 'familyName' [-Wunused-parameter]
warning:     SkTypeface* onMatchFamilyStyleCharacter(const char familyName[], const SkFontStyle& style,
warning:                                                        ^
warning: /Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia/modules/skparagraph/include/TypefaceFontProvider.h:52:89: warning: unused parameter 'style' [-Wunused-parameter]
warning:     SkTypeface* onMatchFamilyStyleCharacter(const char familyName[], const SkFontStyle& style,
warning:                                                                                         ^
warning: /Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia/modules/skparagraph/include/TypefaceFontProvider.h:53:57: warning: unused parameter 'bcp47' [-Wunused-parameter]
warning:                                             const char* bcp47[], int bcp47Count,
warning:                                                         ^
warning: /Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia/modules/skparagraph/include/TypefaceFontProvider.h:53:70: warning: unused parameter 'bcp47Count' [-Wunused-parameter]
warning:                                             const char* bcp47[], int bcp47Count,
warning:                                                                      ^
warning: /Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia/modules/skparagraph/include/TypefaceFontProvider.h:54:55: warning: unused parameter 'character' [-Wunused-parameter]
warning:                                             SkUnichar character) const override {
warning:                                                       ^
warning: /Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia/modules/skparagraph/include/TypefaceFontProvider.h:57:52: warning: unused parameter 'tf' [-Wunused-parameter]
warning:     SkTypeface* onMatchFaceStyle(const SkTypeface* tf, const SkFontStyle& style) const override {
warning:                                                    ^
warning: /Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia/modules/skparagraph/include/TypefaceFontProvider.h:57:75: warning: unused parameter 'style' [-Wunused-parameter]
warning:     SkTypeface* onMatchFaceStyle(const SkTypeface* tf, const SkFontStyle& style) const override {
warning:                                                                           ^
warning: /Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia/modules/skparagraph/include/TypefaceFontProvider.h:61:57: warning: unused parameter 'ttcIndex' [-Wunused-parameter]
warning:     sk_sp<SkTypeface> onMakeFromData(sk_sp<SkData>, int ttcIndex) const override { return nullptr; }
warning:                                                         ^
warning: /Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia/modules/skparagraph/include/TypefaceFontProvider.h:63:49: warning: unused parameter 'ttcIndex' [-Wunused-parameter]
warning:                                             int ttcIndex) const override {
warning:                                                 ^
warning: /Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia/modules/skparagraph/include/TypefaceFontProvider.h:73:49: warning: unused parameter 'path' [-Wunused-parameter]
warning:     sk_sp<SkTypeface> onMakeFromFile(const char path[], int ttcIndex) const override {
warning:                                                 ^
warning: /Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia/modules/skparagraph/include/TypefaceFontProvider.h:73:61: warning: unused parameter 'ttcIndex' [-Wunused-parameter]
warning:     sk_sp<SkTypeface> onMakeFromFile(const char path[], int ttcIndex) const override {
warning:                                                             ^
warning: /Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia/modules/skparagraph/include/TypefaceFontProvider.h:77:55: warning: unused parameter 'familyName' [-Wunused-parameter]
warning:     sk_sp<SkTypeface> onLegacyMakeTypeface(const char familyName[],
warning:                                                       ^
warning: /Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia/modules/skparagraph/include/TypefaceFontProvider.h:78:56: warning: unused parameter 'style' [-Wunused-parameter]
warning:                                            SkFontStyle style) const override {
warning:                                                        ^
warning: 18 warnings generated.
warning: couldn't execute `llvm-config --prefix` (error: No such file or directory (os error 2))
warning: set the LLVM_CONFIG_PATH environment variable to the full path to a valid `llvm-config` executable (including the executable itself)

error: failed to run custom build command for `skia-bindings v0.32.1`

Caused by:
  process didn't exit successfully: `/Users/reportaman/Downloads/neovide/target/release/build/skia-bindings-b5360857fdda4e7f/build-script-build` (exit code: 101)
  --- stdout
  cargo:rerun-if-env-changed=SKIA_DEBUG
  HOST: aarch64-apple-darwin
  cargo:rerun-if-env-changed=SKIA_OFFLINE_SOURCE_DIR
  cargo:rerun-if-env-changed=FORCE_SKIA_BUILD
  cargo:rerun-if-env-changed=FORCE_SKIA_BINARIES_DOWNLOAD
  TRYING TO DOWNLOAD AND INSTALL SKIA BINARIES: 0.32.1/8eb57949962499349b65-aarch64-apple-darwin-textlayout-vulkan
  cargo:rerun-if-env-changed=SKIA_BINARIES_URL
    FROM: https://github.com/rust-skia/skia-binaries/releases/download/0.32.1/skia-binaries-8eb57949962499349b65-aarch64-apple-darwin-textlayout-vulkan.tar.gz
  DOWNLOAD AND INSTALL FAILED: [22] HTTP response code said error (The requested URL returned error: 404 )
  STARTING A FULL BUILD
  Probing 'python'
  Python 2 found: "python"
  Synchronizing Skia dependencies
  Skipping "../src".
  skia/third_party/externals/zlib      @ 47af7c547f8551bd25424e56354a2ae1e9062859
  skia/third_party/externals/harfbuzz  @ 3a74ee528255cc027d84b204a87b5c25e47bff79
  skia/third_party/externals/libpng    @ 386707c6d19b974ca2e3db7f5c61873813c6fe44
  skia/third_party/externals/libjpe... @ 574f3a772c96dc9db2c98ef24706feb3f6dbda9a
  skia/third_party/externals/freetype  @ 0a3d2bb99b45b72e1d45185ab054efa993d97210
  skia/third_party/externals/expat     @ e976867fb57a0cd87e3b0fe05d59e0ed63c6febb
  skia/third_party/externals/icu       @ dbd3825b31041d782c5b504c59dcfb5ac7dda08c
  Skia args: is_official_build=true is_debug=false skia_enable_gpu=true skia_use_gl=false skia_use_system_libjpeg_turbo=false skia_use_system_libpng=false skia_use_libwebp_encode=false skia_use_libwebp_decode=false skia_use_system_zlib=false
skia_use_xps=false skia_use_dng_sdk=false cc="clang" cxx="clang++" skia_use_vulkan=true skia_enable_spirv_validation=false skia_enable_skshaper=true skia_use_icu=true skia_use_system_icu=false skia_use_harfbuzz=true skia_pdf_subset_harfbuzz=true skia_use_system_harfbuzz=false skia_use_sfntly=false skia_enable_skparagraph=true skia_use_expat=true skia_use_system_expat=false
  Done. Made 55 targets from 32 files in 95ms
  ninja: Entering directory `/Users/reportaman/Downloads/neovide/target/release/build/skia-bindings-1943a0b41cb16768/out/skia'
  ninja: no work to do.
  cargo:rerun-if-changed=src/bindings.cpp
  cargo:rerun-if-changed=src/vulkan.cpp
  cargo:rerun-if-changed=src/gpu.cpp
  cargo:rerun-if-changed=src/shaper.cpp
  cargo:rerun-if-changed=src/paragraph.cpp
  cargo:rerun-if-changed=src/svg.cpp
  cargo:rerun-if-changed=/Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia/include
  COMPILING BINDINGS: ["src/bindings.cpp", "src/vulkan.cpp", "src/gpu.cpp", "src/shaper.cpp", "src/paragraph.cpp", "src/svg.cpp"]
  running: "c++" "-O3" "-ffunction-sections" "-fdata-sections" "-fPIC" "-g" "-fno-omit-frame-pointer" "-arch" "arm64" "-I" "/Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia" "-Wall" "-Wextra" "-std=c++17" "-DNDEBUG=" "-DSK_ASSUME_GL=1" "-DSK_ENABLE_API_AVAILABLE=" "-DSK_GAMMA_APPLY_TO_A8=" "-DSK_USE_VMA=" "-DSKIA_IMPLEMENTATION=1" "-DSK_VULKAN=" "-DSK_SUPPORT_PDF=" "-DSK_CODEC_DECODES_JPEG=" "-DSK_ENCODE_JPEG=" "-DSK_CODEC_DECODES_PNG=" "-DSK_ENCODE_PNG=" "-DSK_XML=" "-DSKSHAPER_IMPLEMENTATION=1" "-DSK_SHAPER_HARFBUZZ_AVAILABLE=" "-DU_USING_ICU_NAMESPACE=0" "-DSK_USING_THIRD_PARTY_ICU=" "-o" "/Users/reportaman/Downloads/neovide/target/release/build/skia-bindings-1943a0b41cb16768/out/skia/src/bindings.o" "-c" "src/bindings.cpp"
  exit code: 0
  running: "c++" "-O3" "-ffunction-sections" "-fdata-sections" "-fPIC" "-g" "-fno-omit-frame-pointer" "-arch" "arm64" "-I" "/Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia" "-Wall" "-Wextra" "-std=c++17" "-DNDEBUG=" "-DSK_ASSUME_GL=1" "-DSK_ENABLE_API_AVAILABLE=" "-DSK_GAMMA_APPLY_TO_A8=" "-DSK_USE_VMA=" "-DSKIA_IMPLEMENTATION=1" "-DSK_VULKAN=" "-DSK_SUPPORT_PDF=" "-DSK_CODEC_DECODES_JPEG=" "-DSK_ENCODE_JPEG=" "-DSK_CODEC_DECODES_PNG=" "-DSK_ENCODE_PNG=" "-DSK_XML=" "-DSKSHAPER_IMPLEMENTATION=1" "-DSK_SHAPER_HARFBUZZ_AVAILABLE=" "-DU_USING_ICU_NAMESPACE=0" "-DSK_USING_THIRD_PARTY_ICU=" "-o" "/Users/reportaman/Downloads/neovide/target/release/build/skia-bindings-1943a0b41cb16768/out/skia/src/vulkan.o" "-c" "src/vulkan.cpp"
  exit code: 0
  running: "c++" "-O3" "-ffunction-sections" "-fdata-sections" "-fPIC" "-g" "-fno-omit-frame-pointer" "-arch" "arm64" "-I" "/Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia" "-Wall" "-Wextra" "-std=c++17" "-DNDEBUG=" "-DSK_ASSUME_GL=1" "-DSK_ENABLE_API_AVAILABLE=" "-DSK_GAMMA_APPLY_TO_A8=" "-DSK_USE_VMA=" "-DSKIA_IMPLEMENTATION=1" "-DSK_VULKAN=" "-DSK_SUPPORT_PDF=" "-DSK_CODEC_DECODES_JPEG=" "-DSK_ENCODE_JPEG=" "-DSK_CODEC_DECODES_PNG=" "-DSK_ENCODE_PNG=" "-DSK_XML=" "-DSKSHAPER_IMPLEMENTATION=1" "-DSK_SHAPER_HARFBUZZ_AVAILABLE=" "-DU_USING_ICU_NAMESPACE=0" "-DSK_USING_THIRD_PARTY_ICU=" "-o" "/Users/reportaman/Downloads/neovide/target/release/build/skia-bindings-1943a0b41cb16768/out/skia/src/gpu.o" "-c" "src/gpu.cpp"
  exit code: 0
  running: "c++" "-O3" "-ffunction-sections" "-fdata-sections" "-fPIC" "-g" "-fno-omit-frame-pointer" "-arch" "arm64" "-I" "/Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia" "-Wall" "-Wextra" "-std=c++17" "-DNDEBUG=" "-DSK_ASSUME_GL=1" "-DSK_ENABLE_API_AVAILABLE=" "-DSK_GAMMA_APPLY_TO_A8=" "-DSK_USE_VMA=" "-DSKIA_IMPLEMENTATION=1" "-DSK_VULKAN=" "-DSK_SUPPORT_PDF=" "-DSK_CODEC_DECODES_JPEG=" "-DSK_ENCODE_JPEG=" "-DSK_CODEC_DECODES_PNG=" "-DSK_ENCODE_PNG=" "-DSK_XML=" "-DSKSHAPER_IMPLEMENTATION=1" "-DSK_SHAPER_HARFBUZZ_AVAILABLE=" "-DU_USING_ICU_NAMESPACE=0" "-DSK_USING_THIRD_PARTY_ICU=" "-o" "/Users/reportaman/Downloads/neovide/target/release/build/skia-bindings-1943a0b41cb16768/out/skia/src/shaper.o" "-c" "src/shaper.cpp"
  exit code: 0
  running: "c++" "-O3" "-ffunction-sections" "-fdata-sections" "-fPIC" "-g" "-fno-omit-frame-pointer" "-arch" "arm64" "-I" "/Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia" "-Wall" "-Wextra" "-std=c++17" "-DNDEBUG=" "-DSK_ASSUME_GL=1" "-DSK_ENABLE_API_AVAILABLE=" "-DSK_GAMMA_APPLY_TO_A8=" "-DSK_USE_VMA=" "-DSKIA_IMPLEMENTATION=1" "-DSK_VULKAN=" "-DSK_SUPPORT_PDF=" "-DSK_CODEC_DECODES_JPEG=" "-DSK_ENCODE_JPEG=" "-DSK_CODEC_DECODES_PNG=" "-DSK_ENCODE_PNG=" "-DSK_XML=" "-DSKSHAPER_IMPLEMENTATION=1" "-DSK_SHAPER_HARFBUZZ_AVAILABLE=" "-DU_USING_ICU_NAMESPACE=0" "-DSK_USING_THIRD_PARTY_ICU=" "-o" "/Users/reportaman/Downloads/neovide/target/release/build/skia-bindings-1943a0b41cb16768/out/skia/src/paragraph.o" "-c" "src/paragraph.cpp"
  cargo:warning=In file included from src/paragraph.cpp:10:
  cargo:warning=/Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia/modules/skparagraph/include/ParagraphBuilder.h:19:44: warning: unused parameter 'style' [-Wunused-parameter]
  cargo:warning=    ParagraphBuilder(const ParagraphStyle& style, sk_sp<FontCollection> fontCollection) { }
  cargo:warning=                                           ^
  cargo:warning=/Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia/modules/skparagraph/include/ParagraphBuilder.h:19:73: warning: unused parameter 'fontCollection' [-Wunused-parameter]
  cargo:warning=    ParagraphBuilder(const ParagraphStyle& style, sk_sp<FontCollection> fontCollection) { }
  cargo:warning=                                                                        ^
  cargo:warning=In file included from src/paragraph.cpp:14:
  cargo:warning=/Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia/modules/skparagraph/include/TypefaceFontProvider.h:47:42: warning: unused parameter 'index' [-Wunused-parameter]
  cargo:warning=    SkFontStyleSet* onCreateStyleSet(int index) const override { return nullptr; }
  cargo:warning=                                         ^
  cargo:warning=/Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia/modules/skparagraph/include/TypefaceFontProvider.h:48:47: warning: unused parameter 'familyName' [-Wunused-parameter]
  cargo:warning=    SkTypeface* onMatchFamilyStyle(const char familyName[],
  cargo:warning=                                              ^
  cargo:warning=/Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia/modules/skparagraph/include/TypefaceFontProvider.h:49:55: warning: unused parameter 'style' [-Wunused-parameter]
  cargo:warning=                                   const SkFontStyle& style) const override {
  cargo:warning=                                                      ^
  cargo:warning=/Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia/modules/skparagraph/include/TypefaceFontProvider.h:52:56: warning: unused parameter 'familyName' [-Wunused-parameter]
  cargo:warning=    SkTypeface* onMatchFamilyStyleCharacter(const char familyName[], const SkFontStyle& style,
  cargo:warning=                                                       ^
  cargo:warning=/Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia/modules/skparagraph/include/TypefaceFontProvider.h:52:89: warning: unused parameter 'style' [-Wunused-parameter]
  cargo:warning=    SkTypeface* onMatchFamilyStyleCharacter(const char familyName[], const SkFontStyle& style,
  cargo:warning=                                                                                        ^
  cargo:warning=/Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia/modules/skparagraph/include/TypefaceFontProvider.h:53:57: warning: unused parameter 'bcp47' [-Wunused-parameter]
  cargo:warning=                                            const char* bcp47[], int bcp47Count,
  cargo:warning=                                                        ^
  cargo:warning=/Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia/modules/skparagraph/include/TypefaceFontProvider.h:53:70: warning: unused parameter 'bcp47Count' [-Wunused-parameter]
  cargo:warning=                                            const char* bcp47[], int bcp47Count,
  cargo:warning=                                                                     ^
  cargo:warning=/Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia/modules/skparagraph/include/TypefaceFontProvider.h:54:55: warning: unused parameter 'character' [-Wunused-parameter]
  cargo:warning=                                            SkUnichar character) const override {
  cargo:warning=                                                      ^
  cargo:warning=/Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia/modules/skparagraph/include/TypefaceFontProvider.h:57:52: warning: unused parameter 'tf' [-Wunused-parameter]
  cargo:warning=    SkTypeface* onMatchFaceStyle(const SkTypeface* tf, const SkFontStyle& style) const override {
  cargo:warning=                                                   ^
  cargo:warning=/Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia/modules/skparagraph/include/TypefaceFontProvider.h:57:75: warning: unused parameter 'style' [-Wunused-parameter]
  cargo:warning=    SkTypeface* onMatchFaceStyle(const SkTypeface* tf, const SkFontStyle& style) const override {
  cargo:warning=                                                                          ^
  cargo:warning=/Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia/modules/skparagraph/include/TypefaceFontProvider.h:61:57: warning: unused parameter 'ttcIndex' [-Wunused-parameter]
  cargo:warning=    sk_sp<SkTypeface> onMakeFromData(sk_sp<SkData>, int ttcIndex) const override { return nullptr; }
  cargo:warning=                                                        ^
  cargo:warning=/Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia/modules/skparagraph/include/TypefaceFontProvider.h:63:49: warning: unused parameter 'ttcIndex' [-Wunused-parameter]
  cargo:warning=                                            int ttcIndex) const override {
  cargo:warning=                                                ^
  cargo:warning=/Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia/modules/skparagraph/include/TypefaceFontProvider.h:73:49: warning: unused parameter 'path' [-Wunused-parameter]
  cargo:warning=    sk_sp<SkTypeface> onMakeFromFile(const char path[], int ttcIndex) const override {
  cargo:warning=                                                ^
  cargo:warning=/Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia/modules/skparagraph/include/TypefaceFontProvider.h:73:61: warning: unused parameter 'ttcIndex' [-Wunused-parameter]
  cargo:warning=    sk_sp<SkTypeface> onMakeFromFile(const char path[], int ttcIndex) const override {
  cargo:warning=                                                            ^
  cargo:warning=/Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia/modules/skparagraph/include/TypefaceFontProvider.h:77:55: warning: unused parameter 'familyName' [-Wunused-parameter]
  cargo:warning=    sk_sp<SkTypeface> onLegacyMakeTypeface(const char familyName[],
  cargo:warning=                                                      ^
  cargo:warning=/Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia/modules/skparagraph/include/TypefaceFontProvider.h:78:56: warning: unused parameter 'style' [-Wunused-parameter]
  cargo:warning=                                           SkFontStyle style) const override {
  cargo:warning=                                                       ^
  cargo:warning=18 warnings generated.
  exit code: 0
  running: "c++" "-O3" "-ffunction-sections" "-fdata-sections" "-fPIC" "-g" "-fno-omit-frame-pointer" "-arch" "arm64" "-I" "/Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/skia-bindings-0.32.1/skia" "-Wall" "-Wextra" "-std=c++17" "-DNDEBUG=" "-DSK_ASSUME_GL=1" "-DSK_ENABLE_API_AVAILABLE=" "-DSK_GAMMA_APPLY_TO_A8=" "-DSK_USE_VMA=" "-DSKIA_IMPLEMENTATION=1" "-DSK_VULKAN=" "-DSK_SUPPORT_PDF=" "-DSK_CODEC_DECODES_JPEG=" "-DSK_ENCODE_JPEG=" "-DSK_CODEC_DECODES_PNG=" "-DSK_ENCODE_PNG=" "-DSK_XML=" "-DSKSHAPER_IMPLEMENTATION=1" "-DSK_SHAPER_HARFBUZZ_AVAILABLE=" "-DU_USING_ICU_NAMESPACE=0" "-DSK_USING_THIRD_PARTY_ICU=" "-o" "/Users/reportaman/Downloads/neovide/target/release/build/skia-bindings-1943a0b41cb16768/out/skia/src/svg.o" "-c" "src/svg.cpp"
  exit code: 0
  running: "ar" "cq" "/Users/reportaman/Downloads/neovide/target/release/build/skia-bindings-1943a0b41cb16768/out/skia/libskia-bindings.a" "/Users/reportaman/Downloads/neovide/target/release/build/skia-bindings-1943a0b41cb16768/out/skia/src/bindings.o" "/Users/reportaman/Downloads/neovide/target/release/build/skia-bindings-1943a0b41cb16768/out/skia/src/vulkan.o" "/Users/reportaman/Downloads/neovide/target/release/build/skia-bindings-1943a0b41cb16768/out/skia/src/gpu.o" "/Users/reportaman/Downloads/neovide/target/release/build/skia-bindings-1943a0b41cb16768/out/skia/src/shaper.o" "/Users/reportaman/Downloads/neovide/target/release/build/skia-bindings-1943a0b41cb16768/out/skia/src/paragraph.o" "/Users/reportaman/Downloads/neovide/target/release/build/skia-bindings-1943a0b41cb16768/out/skia/src/svg.o"
  exit code: 0
  running: "ar" "s" "/Users/reportaman/Downloads/neovide/target/release/build/skia-bindings-1943a0b41cb16768/out/skia/libskia-bindings.a"
  exit code: 0
  GENERATING BINDINGS
  cargo:warning=couldn't execute `llvm-config --prefix` (error: No such file or directory (os error 2))
  cargo:warning=set the LLVM_CONFIG_PATH environment variable to the full path to a valid `llvm-config` executable (including the executable itself)

  --- stderr
  Apple clang version 12.0.0 (clang-1200.0.32.29)
  Target: aarch64-apple-darwin
  Thread model: posix
  InstalledDir: /Library/Developer/CommandLineTools/usr/bin
  ignoring nonexistent directory "/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/c++/v1"
  thread 'main' panicked at 'libclang error; possible causes include:
  - Invalid flag syntax
  - Unrecognized flags
  - Invalid flag arguments
  - File I/O errors
  - Host vs. target architecture mismatch
  If you encounter an error missing from this list, please file an issue or a PR!', /Users/reportaman/.cargo/registry/src/github.com-1ecc6299db9ec823/bindgen-0.54.0/src/ir/context.rs:573:15
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
>>
```

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -79,10 +79,15 @@ jobs:
         mv MoltenVK_icd.json ./Resources/vulkan/icd.d/
         install_name_tool -add_rpath "@executable_path/../Frameworks" ./MacOS/neovide
 
+    - name: Create .dmg file
+      run: |
+        hdiutil create Neovide-uncompressed.dmg -volname "Neovide" -srcfolder target/release/bundle/osx
+        hdiutil convert Neovide-uncompressed.dmg -format UDZO -o Neovide.dmg
+
     - uses: actions/upload-artifact@v1
       with:
-        name: Neovide.app
-        path: ./target/release/bundle/osx
+        name: Neovide.dmg
+        path: ./Neovide.dmg
 
   build-linux:
 

